release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -2476,6 +2476,65 @@ index a90ab14ff..f7f6f1a71 100644
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=2097579[*BZ#2097579*])
 
+[id="ocp-4-11-ran-known-issues"]
+== {product-title} 4.11 RAN known issues
+
+* After deploying a {sno} with the RAN DU profile, an error `openvswitch: cpu_id mismatch with handler threads` is continuously generating in the Open vSwitch kernel logs. (link:https://issues.redhat.com/browse/OCPBUGSM-46165[*OCPBUGSM-46165*])
+
+* If secure boot is currently disabled and you attempt to enable it through ZTP, the cluster installation does not start. When secure boot is enabled through ZTP, the boot options are configured before the virtual CD is attached. Therefore, the first boot is from the existing hard disk with secure boot turned on. The cluster installation gets stuck because the system never boots from the CD. (link:https://issues.redhat.com/browse/OCPBUGSM-45085[*OCPBUGSM-45085*])
+
+* If an invalid subscription channel is specified in the subscription policy that is used to perform a cluster upgrade, the {cgu-operator-full} indicates a successful upgrade right after the policy is enforced because the state does not change (`AtLatestKnown`). (link:https://issues.redhat.com/browse/OCPBUGSM-43618[*OCPBUGSM-43618*])
+
+* If SRIOV-FEC policies are added separately, the {cgu-operator} adds the policies to the top of the policy list. The original content in the `/source-crs` folder overrides the contents updated by the enforce policy, which causes the original inform policy to never become `Compliant`. The workaround is to add the `CatalogSource` change in the `common-config-policy`. (link:https://issues.redhat.com/browse/OCPBUGSM-39859[*OCPBUGSM-39859*])
```

Comment:
```suggestion
* If installation of the SRIOV-FEC operator is done through separate policies and the CatalogSource reuses the name of a default catalog source the operator installation may fail due to conflicts with the management of the default catalog sources. To avoid this issue the SRIOV-FEC CatalogSource should be added to the `common-config-policy` and the operator subscription should be added to the `common-subscriptions-policy`. The workaround when using separate policies to install the SRIOV-FEC operator is to ensure the CatalogSource for this operator is uniquely named. (link:https://issues.redhat.com/browse/OCPBUGSM-39859[*OCPBUGSM-39859*])
```